content/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -124,17 +152,180 @@ <h3>Service 和 Label</h3>
 		<br>
 		<div class="row">
 			<div class="col-md-8">
-<!--        <p>Labels can be attached to objects at creation time or later on. They can be modified at any time. Let's expose our application now using a Service and apply some labels.</p>-->
-        <p> 标签(Label)可以在创建时或之后附加到对象上。他们可以随时被修改。现在使用 Service 发布我们的应用程序并添加一些 Label 。</p>
+				<!--
+				<p>Labels can be attached to objects at creation time or later on. They can be modified at any time. Let's expose our application now using a Service and apply some labels.</p>
+				-->
+				<p> 标签（Label）可以在创建时或之后附加到对象上。他们可以随时被修改。现在使用 Service 发布我们的应用程序并添加一些标签。</p>
```

Comment:
只有第一次提及某术语时需要加英文括号，后面就不必要再加了。

```suggestion
				<p>标签可以在对象创建时或之后附加到对象上。它们可以随时被修改。现在使用 Service 发布我们的应用程序并添加一些标签。</p>
```